django/db/backends/base/schema.py

```diff
@@ -1999,6 +2006,11 @@ def _constraint_names(
                     result.append(name)
         return result
 
+    def _pk_constraint_sql(self, columns):
+        return self.sql_pk_constraint % {
+            "columns": ", ".join(self.quote_name(column) for column in columns)
+        }
+
```

Comment:
I think that might have been an opportunity to always use the `PRIMARY KEY` constraint syntax even for single column primary keys (instead of the `PRIMARY KEY` next to the column) but that should do for now.

Reply:
We could do a refactoring later, I didn't want to deal with it in this PR. I'm not sure if it's worth changing